pypeit/spectrographs/mdm_modspec.py

```diff
@@ -0,0 +1,317 @@
+    
+        datasec = np.atleast_1d([
+            '[{0:d}:{1:d},{2:d}:{3:d}]'.format(1, lenSpec, 1, 300)
+        ])
+        oscansec = np.atleast_1d([
+            '[{0:d}:{1:d},{2:d}:{3:d}]'.format(1, lenSpec, 308, lenSpat)
+
+        ])
+        if hdu is None:
+            binning = '1,1'                 # Most common use mode
+        else:
+            binning = "{0},{1}".format(
+                hdu[0].header['CCDBIN1'], hdu[0].header['CCDBIN2'])
+
+        # Detector 1 continued
+        detector_dict = dict(
+            binning         = binning,
+            det             = 1,
+            dataext         = 0,
+            specaxis        = 0,        # Native spectrum is along the x-axis 
+            specflip        = True,     # Wavelength decreases as pixel number increases
+            spatflip        = False,    # Spatial position increases as pixel number increases
+            platescale      = 0.28,     # Arcsec / pixel
+            darkcurr        = 0.0,      # Electrons per hour
+            saturation      = 65535.,   # 16-bit ADC
+            nonlinear       = 0.97,     # Linear to ~97% of saturation
+            mincounts       = -1e10,
+            numamplifiers   = 1,
+            gain            = gain,     # See above
+            ronoise         = ronoise,  # See above
+            # Data & Overscan Sections -- Edge tracing can handle slit edges
+            datasec         = datasec,  # See above
+            oscansec        = oscansec  # See above
+            )
+        # Return
+        return detector_container.DetectorContainer(**detector_dict)
+        
+    @classmethod
+
+    def default_pypeit_par(cls):
+        """
+        Return the default parameters to use for this instrument.
+        
+        Returns:
+            :class:`~pypeit.par.pypeitpar.PypeItPar`: Parameters required by
+            all of ``PypeIt`` methods.
+        """
+        par = super().default_pypeit_par()
+
+        # Slit edge method
+        par['calibrations']['slitedges']['sync_predict'] = 'nearest' # Ignore PCA
+        par['calibrations']['slitedges']['bound_detector'] = True # Edges of slit fall off the detector, so assign edges of detector as the edges of the slit
+
+        # Pixel flat method
+        par['calibrations']['pixelflatframe']['process']['combine'] = 'mean'
+        par['calibrations']['pixelflatframe']['process']['clip'] = True
+        par['calibrations']['pixelflatframe']['process']['comb_sigrej'] = 3.0 
+        par['calibrations']['pixelflatframe']['process']['n_lohi'] = [1, 1] 
+        par['calibrations']['pixelflatframe']['process']['use_overscan'] = True  
+        
+        # Wavelength calibration method
+        par['calibrations']['wavelengths']['method'] = 'full_template' 
+        par['calibrations']['wavelengths']['lamps'] = ['ArI', 'XeI', 'NeI']
+        par['calibrations']['wavelengths']['reid_arxiv'] = 'mdm_modspec_1200_5100.fits'
```

Comment:
Is there only one mode? Is it worthwhile putting this in a `config_specific_par`?

Reply:
only one so far :)

I bet only one going forth too.